openshift/logstash.dc.yaml

```diff
@@ -458,6 +458,11 @@ parameters:
     displayName: Target Namespace
     required: true
     value: ixhmbm-dev
+  - name: HOST_ROUTE
+    description: The host the route will use to expose service outside cluster
+    displayName: Host route
+    required: true
+    value: logstash-dev.pathfinder.gov.bc.ca
```

Comment:
If logstash and clogs are to be in different namespace than the main elasticsearch and kibana, I don't know if we should be changing the default host route to something like this without the namespace in it as there could be other logstashes that may exist down the line.

Reply:
We will deal with that when we carve out Logstash and CLOGS.  Having the namespace in the route makes the code less portable and more effort to move logstash.